dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -195,123 +193,123 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
-			            Logger.error(this, "Error: Unable to determine if there's a logged user.", e);
-			        }
-			    }
-			    //GIT-4506
-			    if(WebAPILocator.getUserWebAPI().isLoggedToBackend(req)){
-			        isLiveMode = (!EDIT_MODE && !PREVIEW_MODE);
-			    }
-
-			    if (req.getSession(false) != null && req.getSession().getAttribute("tm_date")!=null) {
-			        isLiveMode = Boolean.TRUE;
-			        Identifier ident=APILocator.getIdentifierAPI().find(assetIdentifier);
-			        if(UtilMethods.isSet(ident.getSysPublishDate()) || UtilMethods.isSet(ident.getSysExpireDate())) {
-			            Date fdate = new Date(Long.parseLong((String)req.getSession().getAttribute("tm_date")));
-			            isLiveMode = ifIdentifierIsUnpublishedOrExpired(ident.getSysPublishDate(),fdate);
-			            isContentExpired = ifIdentifierIsUnpublishedOrExpired(ident.getSysExpireDate(),fdate);
-			        }
-                    if(isContentExpired){
-                        Logger.debug(this, "Id " + assetIdentifier + " belongs to an expired content.");
-                        resp.sendError(HttpServletResponse.SC_NOT_FOUND);
-                        return;
-                    }
-			    }
-
-			    //If the DEFAULT_FILE_TO_DEFAULT_LANGUAGE is true and the default language is NOT equals to the language we have in request/session...
-			    if ( Config.getBooleanProperty("DEFAULT_FILE_TO_DEFAULT_LANGUAGE", false)
-			            && defaultLang != lang ) {
-
-			        ContentletAPI contentletAPI = APILocator.getContentletAPI();
-
-			        //Build the lucene query with the identifier and both languages, the default and one in session to see what we can find
-			        StringBuilder query = new StringBuilder();
-			        query.append("+(languageId:").append(defaultLang).append(" languageId:").append(lang).append(") ");
-			        query.append("+identifier:").append(assetIdentifier).append(" +deleted:false ");
-			        
-			        query.append((isLiveMode) ? "+live:true ": "+working:true ");
-
-			        List<Contentlet> foundContentlets = contentletAPI.search(query.toString(), 2, -1, null, user, respectFrontendRoles);
-			        if ( foundContentlets != null && !foundContentlets.isEmpty() ) {
-			            //Prefer the contentlet with the session language
-			            content = foundContentlets.get(0);
-			            if ( content.getLanguageId() != lang && foundContentlets.size() == 2 ) {
-			                content = foundContentlets.get(1);
-			            }
-			        } else {
-			            Logger.error(this, "Content with Identifier [" + assetIdentifier + "] not found.");
-			            resp.sendError(HttpServletResponse.SC_NOT_FOUND);
-			            return;
-			        }
-
-			    }
-			    else {
-			        /*
+
+			if (isContent){
+				Contentlet content = null;
+				if(byInode) {
+					if(isTempBinaryImage)
+						content = contentAPI.find(assetInode, APILocator.getUserAPI().getSystemUser(), respectFrontendRoles);

```

Comment:
this could be 

content = (isTempBinaryImage)?
contentAPI.find(assetInode, APILocator.getUserAPI().getSystemUser(), respectFrontendRoles):
contentAPI.find(assetInode, user, respectFrontendRoles);